docs/source/nlp/megatron_onnx_export.rst

```diff
@@ -0,0 +1,56 @@
+.. _megatron_onnx_export:
+
+ONNX Export of Megatron Models
+====================================
+
+This guide demonstrates the usage of the ONNX export functionality for Megatron models.
+
+Installation
+-----------------
+Set up the development environment by launching the latest NeMo container
+
+.. code-block:: bash 
+
+    docker run -it nvcr.io/nvidia/nemo:23.04
+
+ONNX Export of Megatron Models requires the installation of the latest versions of NeMo and TransformerEngine. The snippet below can be used to perform both installations
+
+.. code-block:: bash 
+
+    https://github.com/NVIDIA/TransformerEngine.git && cd TransformerEngine
+    git submodule update --init --recursive
+    python3 -m pip install -e .
+
+    cd ..
+    git clone https://github.com/NVIDIA/NeMo.git && cd NeMo/
+    python3 -m pip install -e .
```

Comment:
We have installation instructions already. Maybe you can just mention a tagged minimum version of TE and NeMo that is required?

Reply:
Done